docs/sources/dashboards/create-reports/_index.md

```diff
@@ -0,0 +1,331 @@
+- [Embed a dashboard image in the email](#embed-a-dashboard-as-an-image-in-the-email)
+- [Attach a CSV file of the table panel data](#attach-a-csv-file-of-the-table-panel-data)
+- [Attach a separate PDF of table data](#table-data-in-pdf) (Public preview only)
+
+#### Attach the report as a PDF
+
+If you selected the PDF format option, under the **Style the PDF** section, set the following options:
+
+- **Configure the report header** - Click the checkbox to select this option.
+- **Orientation** - Set the report orientation in **Portrait** or **Landscape**.
+- **Layout** - Select one of the following:
+  - **Simple** - Renders each panel as full-width across the PDF. Refer to the [Layout and orientation table](#layout-and-orientation) to see examples.
+  - **Grid** - Renders the PDF with the same panel arrangement and width as the source dashboard. Refer to the [Layout and orientation table](#layout-and-orientation) to see examples.
+- **Zoom** - Zoom in to enlarge text in your PDF, or zoom out to see more data (like table columns) per panel.
+
+Click **Preview PDF** in the top-right corner of the screen to view a rendered PDF with the options you selected.
+
+##### Layout and orientation
+
+<!-- prettier-ignore-start -->
+
+| Layout | Orientation | Description                                                                                               | Preview    |
+| ------ | ----------- | --------------------------------------------------------------------------------------------------------- | ------------ |
+| Simple | Portrait    | Generates an A4 page in portrait mode with three panels per page.                                         | {{< figure src="/static/img/docs/enterprise/reports_portrait_preview.png" max-width="500px" alt="Simple layout in portrait" >}}    |
+| Simple | Landscape   | Generates an A4 page in landscape mode with a single panel per page.                                      | {{< figure src="/static/img/docs/enterprise/reports_landscape_preview.png" max-width="500px" alt="Simple layout in landscape" >}}                     |
+| Grid   | Portrait    | Generates an A4 page in portrait mode with panels arranged in the same way as at the original dashboard.  | {{< figure src="/static/img/docs/enterprise/reports_grid_portrait_preview.png" max-width="500px" alt="Grid layout in portrait" >}} |
+| Grid   | Landscape   | Generates an A4 page in landscape mode with panels arranged in the same way as in the original dashboard. | {{< figure src="/static/img/docs/enterprise/reports_grid_landscape_preview.png" max-width="500px" alt="Grid layout in landscape" >}}                  |
+
+<!-- prettier-ignore-end -->
+
+#### Embed a dashboard as an image in the email
+
+You can send a report email with an image of the dashboard embedded in the email instead of attached as a PDF. In this case, the email recipients can see the dashboard at a glance instead of having to open the PDF.
```

Comment:
`instead of` suggest that it's one or the other while the user can select both PDF and image and will receive the embedded image with a PDF attachment.